tools/query_verification/src/main/java/com/google/bigquery/QueryVerification.java

```diff
@@ -0,0 +1,85 @@
+package com.google.bigquery;
+
+public class QueryVerification {
+
+    private QVQuery migratedQuery;
+    private QVSchema migratedSchema;
+
+    private QVQuery originalQuery;
+    private QVSchema originalSchema;
+
+    public QueryVerification() {
+    }
+
+    public QueryVerification(QVQuery migratedQuery, QVSchema migratedSchema) {
+        this.migratedQuery = migratedQuery;
+        this.migratedSchema = migratedSchema;
+    }
+
+    public QueryVerification(QVQuery migratedQuery, QVSchema migratedSchema, QVQuery originalQuery, QVSchema originalSchema) {
```

Comment:
Will it be better to have a single constructor and mark the input params as @Nuallable